src/customers/controller/customers.controller.ts

```diff
@@ -22,6 +28,8 @@ export class CustomersController {
   @ApiResponse({ status: 200, description: 'The user was successfully found' })
   @ApiResponse({ status: 404, description: "The user can't be found" })
   @Get(':id')
+  @UseGuards(RolesGuard)
+  @Roles(['admin'])
```

Comment:
You should make use of the `CustomerRole` here: `@Roles([CustomerRole.ADMIN])`.

Instead of the magic string, we can use the enum. 

Using magic strings: 
- prone to typos.
- refactoring is error-prone
- it’s harder to manage roles consistently across the application

Reply:
done